src/main/java/hw_6/Task1.java

```diff
@@ -0,0 +1,42 @@
+package hw_6;
+
+public class Task1 {
+    public static void main(String[] args) {
+
+        Task1 task = new Task1();
+        task.NonStaticMethod();
+
+        StaticMethod();
+
+    }
```

Comment:
Task1 Should be a separate class. So, please create a separate App class for main method and move this main method there

Reply:
fixed